### PR TITLE
Remove old refs to outdated Mayadata info

### PR DESCRIPTION
**Pull request by orville-wright**
Remove old Mayadata irrelevant info
Signed-off-by: Orville Wright <orville.wright@yahoo.com>

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -65,82 +65,63 @@ Once review is given, Maintainers and SIG members may indicate merge readiness w
...
-- [@GlennBullingham][members-glennbullingham] - [@mayadata-io][maya-data]
+> [!Important]
+> When creating an ISSUE or a PR. Please TAG one (or more) of the Maintainers.
 
 Our Special Interest Groups (SIGs) are:
+| Functional area | Name | @GitHub | Are of Expertese | Product specialization |
+| :--- | :--- | :--- | :--- | :--- |
+| ``` Entire Product ``` | Tiago Castro | @tiagolobocastro | Architect / Eng | Mayastor |
+| ``` Data Plane ``` | Dimitry Savitskiy | @dsavitskiy | Architect / Eng | Mayastor |
+| ``` Data Plane ``` | Jonathan Teh | @jonathan-teh | Eng | Mayastor |
```
Comment (chriswldenyer) [inline CONTRIBUTING.md:95]: Jonathan Teh, Mikhail Tcymbaliuk and Paul Yoong are no longer maintainers AFAIK.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -65,82 +65,63 @@ Once review is given, Maintainers and SIG members may indicate merge readiness w
...
+| ``` Data Plane ``` | Mikhail Tcymbaliuk | @mtzaurus | Archietct / Eng | Mayastor |
+| ``` Control Plane ``` | Paul Yoong | @paulyoong | Archietct / Eng | Mayastor |
+| ``` 2e2 Testing ``` | Blaise Dias | @blaisedias | Eng / Test / QA | All products |
```
Comment (chriswldenyer) [inline CONTRIBUTING.md:98]: e2e here ?

Comment (orville-wright) [reply CONTRIBUTING.md:98]: We just need to show it somewhere for now.

File: CONTRIBUTING.md (hunk 1 above)
Comment (orville-wright) [reply CONTRIBUTING.md:95]: Yup, I know. The Maintainers need to decide that to do with the names. For now, we just need remove all the very old Mayadata refs.

**Review by edrob999 — APPROVED** [review]
:thumbsup:
Signed-off-by: Ed Robinson <ed.robinson@gmail.com>

File: CONTRIBUTING.md (hunk 2 above)
Comment (chriswldenyer) [reply CONTRIBUTING.md:98]: I just meant "2e2" should be "e2e"

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -65,82 +65,63 @@ Once review is given, Maintainers and SIG members may indicate merge readiness w
...
+<BR>
 
-- [@gila][members-gila] - [@mayadata-io][maya-data]
-- [@jkryl][members-jkryl] - [@mayadata-io][maya-data]
-- [@GlennBullingham][members-glennbullingham] - [@mayadata-io][maya-data]
+> [!Important]
+> When creating an ISSUE or a PR. Please TAG one (or more) of the Maintainers.
 
 Our Special Interest Groups (SIGs) are:
+| Functional area | Name | @GitHub | Are of Expertese | Product specialization |
```
Comment (dsharma-dc) [inline CONTRIBUTING.md]:
```suggestion
| Functional area | Name | @GitHub | Area of Expertise | Product specialization |
```

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -65,82 +65,63 @@ Once review is given, Maintainers and SIG members may indicate merge readiness w
...
+| :--- | :--- | :--- | :--- | :--- |
+| ``` Entire Product ``` | Tiago Castro | @tiagolobocastro | Architect / Eng | Mayastor |
+| ``` Data Plane ``` | Dimitry Savitskiy | @dsavitskiy | Architect / Eng | Mayastor |
```
Comment (dsharma-dc) [inline CONTRIBUTING.md]:
```suggestion
| ``` Data Plane ``` | Dmitry Savitskiy | @dsavitskiy | Architect / Eng | Mayastor |
```

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -65,82 +65,63 @@ Once review is given, Maintainers and SIG members may indicate merge readiness w
...
+| ``` Data Plane ``` | Jonathan Teh | @jonathan-teh | Eng | Mayastor |
+| ``` Data Plane ``` | Mikhail Tcymbaliuk | @mtzaurus | Archietct / Eng | Mayastor |
+| ``` Control Plane ``` | Paul Yoong | @paulyoong | Archietct / Eng | Mayastor |
+| ``` 2e2 Testing ``` | Blaise Dias | @blaisedias | Eng / Test / QA | All products | 
+| ``` 2e2 Testing ``` | Chris Denyer | @chriswldenyer | Eng / QA | Mayastor |
```
Comment (dsharma-dc) [inline CONTRIBUTING.md]:
```suggestion
| ``` e2e Testing ``` | Chris Denyer | @chriswldenyer | Eng / QA | Mayastor |
```

**Review by chriswldenyer — APPROVED** [review]

**Review by dsharma-dc — APPROVED** [review]
Please squash the commits into a single commit and then merge.

**Pull request closed without merging**